Gemfile.lock

```diff
@@ -1,12 +1,12 @@
 GIT
-  remote: git://github.com/everypolitician/everypolitician-ruby.git
+  remote: https://github.com/everypolitician/everypolitician-ruby.git
```

Comment:
So this change to `Gemfile.lock` was now generated by bundler instead of by me :)